client/src/components/Button/Buttons.module.scss

```diff
@@ -31,3 +31,30 @@
     }
   }
 }
+
+/* -------------------------------------------------------------------------- */
+/*                                PrimaryButton                               */
+/* -------------------------------------------------------------------------- */
+
+.primaryButton {
```

Comment:
Est-ce qu'on grossirait pas un petit poil ce bouton pour le faire ressortir un peu mieux ?

Reply:
> Est-ce qu'on grossirait pas un petit poil ce bouton pour le faire ressortir un peu mieux ?

Ça marche 👍

Reply:
J'arrive un peu après la guerre. Je me dis si on veut vraiment faire ressortir le bouton c'est d'alterner le style hover (background vert/text white) avec l'état normal. Je sais pas ce que vous en pensez ?

Reply:
Ah voile j'ai retrouvé la discussion. Voici le rendu avec les deux options. Perso j'ai une petite préférence pour le bouton vert aussi comme Jeff, parce que ça fait un rappel au header et que ça fait plus ressortir le bouton. @Lisadev84 tu en penses quoi ? Si on est 2 contre 2 faudra tirer au sort ! xD

<img width="1677" alt="Capture d’écran 2024-11-18 à 12 24 34" src="https://github.com/user-attachments/assets/c3429e95-cdb1-421b-aa0c-6c00f3c5d8fe">

<img width="1677" alt="Capture d’écran 2024-11-18 à 12 25 27" src="https://github.com/user-attachments/assets/609acb29-6153-4e8f-a267-e461f255e342">

Reply:
J'avoue préférer la version d'Ilyas moins agressive mais s'agissant d'un call To Action, il est préférable de privilégier la visibilité et le contraste avec un background vert. Peut-être adoucir un peu le vert.